lib/drivers/tandemDriver.js

```diff
@@ -1595,6 +1595,14 @@ module.exports = function (config) {
           debug('Event with unhandled change type:', event);
           throw new Error('Unhandled combination of basal change types: ' + event.change_types.join('|'));
       }
+      if(postbasal.deliveryType === 'temp') {
```

Comment:
Ah, it looks like that happens when there's a pump shutdown event. I've moved the logic inside, thanks.